server/main.go

```diff
@@ -103,6 +104,10 @@ func main() {
 		glog.Fatalln("Cannot config global logger, use default one: ", zap.Error(err))
 	}
 
+	// check device
+	stopWatch := watch.Start(ctx, 5*time.Minute)
+	defer stopWatch()
```

Comment:
just call `watch.Start()` in `server.Start()`, and call `watch.Stop()` in `server.Stop()`?

Reply:
Just check like heartbeat.  `server` contains the major components like api blockchain etc...